cleaners/internet_explorer.xml

```diff
@@ -40,20 +50,92 @@
+    <action command="delete" search="walk.all" path="%AppData%\Microsoft\Windows\IECompat*Cache\"/>
+    <action command="delete" search="walk.files" path="%LocalAppData%\Microsoft\Windows\INetCache\"/>
+    <action command="delete" search="walk.files" path="%LocalAppData%\Microsoft\Windows\WebCache\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Microsoft\Windows\WebCache.old\"/>
+    <action command="delete" search="glob" path="%LocalAppData%\Microsoft\Windows\WebCache.old\"/>
+    <!-- Extras - making WinApp2.ini obsolete: -->
+    <action command="delete" search="walk.all" path="%LocalAppData%\Microsoft\Windows\AppCache\"/>
+    <action command="delete" search="glob" path="%LocalAppData%\Microsoft\Windows\AppCache\*"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\AppCache\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\IECompat*Cache\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\INet*\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\Microsoft\Internet Explorer\DOMStore\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\Microsoft\Internet Explorer\Emie*List\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\PRICache\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\AC\Temp\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\LocalState\Cache\"/>
+    <action command="delete" search="walk.all" path="%LocalAppData%\Packages\windows_ie_ac_*\TempState\"/>
+    <action command="delete" search="walk.all" path="%WinDir%\System32\config\Systemprofile\AppData\Local\Microsoft\Windows\INetCache\"/>
+    <action command="winreg" path="HKCR\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppContainer\Storage\windows_ie_ac_001\Internet Explorer\DOMStorage"/>
+    <action command="winreg" path="HKCU\Software\Microsoft\Internet Explorer\LowRegistry\DOMStorage"/>
+    <!-- Used e.g. in Windows XP (and maybe before on NT) - English-only-Support: -->
+    <!-- Was, and didn't worked: -->
+    <!-- <action command="delete" search="walk.files" path="%USERPROFILE%\Local Settings\Temporary Internet Files\"/> -->
+    <action command="delete" search="walk.all" path="%USERPROFILE%\Local Settings\Temporary Internet Files\"/>
+    <action command="delete" search="glob" path="%USERPROFILE%\Local Settings\Temporary Internet Files\*"/>
+    <!-- Used e.g. in Windows XP (and maybe before on NT) - some none-English-Support: -->
+    <!-- Polish Windows XP Support: -->
+    <action command="delete" search="walk.all" path="%USERPROFILE%\Ustawienia lokalne\Temporary Internet Files\"/>
```

Comment:
I could merge the Polish parts for BleachBit version 2.1/2.2 if it were a separate pull request (it's up to you). This is a small change, and the bug reporter confirmed the fix.

There are some big changes in the whole pull request to an important application (MSIE), so the rest will need to wait for version 2.3/2.4. Thanks for your patience

Reply:
Here is a PR for Polish Windows XP Support: https://github.com/bleachbit/bleachbit/pull/485

Reply:
Maybe you should also merge https://github.com/bleachbit/bleachbit/pull/475 for BB2.2... it's also Polish support... It got confirmed here: https://github.com/bleachbit/bleachbit/issues/460#issuecomment-471987298